### DISCUSSION TITLE
Error producing PDF. ! Undefined control sequence. l.22 \hypertarget

### DISCUSSION THREAD

**Discussion by Maurdramnus** [Zettlr/Zettlr#4280] [Q&A]
Hi

I'm new to Zettlr and very impressed. 
I get the following error when compiling to pdf(Latex):

Error producing PDF.
! Undefined control sequence.
l.22 \hypertarget

I have tried removing citations and headlines but still get the error. 
Does anyone know how to solve this?

**Comment by nathanlesage** [comment]
\hypertarget is a command defined by some package that you need to install first.

You'll need to search CTAN for that command. It will tell you the name of the package that you need to install. Afterwards it should work fine.

I'll add a section on how to do this in the new docs for 3.0
